hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OFSPath.java

```diff
@@ -63,16 +66,19 @@ public class OFSPath {
   private String bucketName = "";
   private String mountName = "";
   private String keyName = "";
+  private OzoneConfiguration conf;
   private static final String OFS_MOUNT_NAME_TMP = "tmp";
   // Hard-code the volume name to tmp for the first implementation
   @VisibleForTesting
   public static final String OFS_MOUNT_TMP_VOLUMENAME = "tmp";
 
-  public OFSPath(Path path) {
```

Comment:
One way to avoid changing every single existing usage is to keep the old constructor:

```
  public OFSPath(Path path) {
    this.conf = new OzoneConfiguration();
    initOFSPath(path.toUri(), false);
  }
```

Or even just `this.conf = null;`, given proper handling (null check).

Reply:
As discussed offline this constructor change should be fine since the interface is `Unstable` and `Private`.

As long as the new Ozone client (after an upgrade) would use the corresponding new `ozone-common` jar it should be fine.